docs/development/local_setup.md

```diff
@@ -320,36 +333,41 @@ $ vagrant destroy --force
 ==> core-01: Forcing shutdown of VM...
 ==> core-01: Destroying VM and associated drives...
 
-$ cd $GOPATH/src/github.com/gardener/gardener
+$ cd ..
 ```
 
-#### Start the Gardener Local Provider
-
-The Seed cluster provides the possibility to create Shoot clusters on several cloud provider. The Gardener Provider implements a common interface to all supported cloud providers. Here, the corresponding Gardener Provider for Local is used.
+#### Create, access and delete a Shoot Cluster
 
-By executing
+Now, you can create a Shoot cluster by running
 
 ```bash
-$ make start-local
-2018/02/14 10:53:34 Listening on :3777
-2018/02/14 10:53:34 Vagrant directory /Users/foo/go/src/github.com/gardener/gardener/vagrant
-2018/02/14 10:53:34 user-data path /Users/foo/git/go/src/github.com/gardener/gardener/dev/user-data
+$ kubectl apply -f dev/90-shoot-local.yaml
+shoot "local" created
 ```
 
-the Gardener Local Provider is started.
+Wait until the 2 secrets `osc-result-cloud-config-local-*` appear in the Shoot cluster namespace and then copy `cloud_config` from the secret `osc-result-cloud-config-local-xxxxx-downloader` to the local file `dev/user-data`:
```

Comment:
I was wondering how does the vagrant VM join the shoot cluster. It looks like that the `dev/user-data` file is actually used for this purpose. Previously the file was generated by the Local(CloudProvider/Botanist), but now it is not.

Could you write a few words with motivation/description how/why the `user-data` file is used.
You could do it here or a few lines below just before `vagrant up`. I just want the documentation to be clear why we need to download the cloud config from a secret and save it in a file `dev/user-data`.

Reply:
I will add a sentence about that.